src/Shared/AppModels/AppRoot.cs

```diff
@@ -373,6 +389,15 @@ private async Task HandleCoreSettingsCollectionChangedAsync<TSettings>(object? s
                         Logger.LogInformation($"Adding new core {newCore.DisplayName}, {nameof(newCore.InstanceId)} {newCore.InstanceId}");
                         _mergedCore.AddSource(newCore);
                     }
+
+                    Guard.IsNotNull(Ipfs);
+                    var mergedCoreWithPlugins = CreatePluginLayer(_mergedCore, Ipfs);
+                    StrixDataRoot = new StrixDataRootViewModel(mergedCoreWithPlugins);
```

Comment:
There's no need to do any of this new stuff. This method calls `InitAsync` again, as it has all the code to do this already and may do other things needed for setup. See [here](https://github.com/Arlodotexe/strix-music/blob/8ca4df16eba5a844704759ab8bf3475148487c45/src/Shared/AppModels/AppRoot.cs#L185).